packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -667,7 +668,7 @@ export default function myPostForm(props) {
           setModelFields(onSubmitBefore({ fields: modelFields }));
         }
         try {
-          const original = await DataStore.query(Post, id);
+          const original = id ? await DataStore.query(Post, id) : currentModel;
```

Comment:
so i think we need to move the query up since we need to provide the defaultValues on the form as well

Reply:
Got it